### ISSUE TITLE
Clear selection after finished transfer

### ISSUE THREAD

**Issue by 601958483** [localsend/localsend#165]
### Is your feature request related to a problem? Please describe.

我目前主要使用场景是Android端互传PC端文件。为了便捷，PC端会长期自启动留驻后台，传完一个文件后发现还需要传另外一个，但是这时候就需要先把之前传输完成的文件手动删除，不然添加了新文件传输，旧文件也会一起再被传输

### Describe the solution you'd like

希望可以添加一个，对同一设备传输的文件的去重复功能

**Comment by Tienisto** [comment]
I couldn't understand your request completely, but I can add an option to overwrite the duplicate.

**Comment by rzhangbq** [comment]
I know what @601958483 means. @601958483 wants the file list to be emptied automatically once the transfer is finished.

**Comment by Tienisto** [comment]
Thanks, I am closing this because this issue has already been mentioned.

Duplicate of #147
